.github/workflows/_checkmarx.yml

```
@@ -0,0 +1,68 @@
+name: Perform Checkmarx Scan
```

```suggestion
name: Scan with Checkmarx
```
Action-oriented statements with names.